docs/docs/guides/creating-your-own-database-connector.md

```diff
@@ -0,0 +1,226 @@
+# Creating A Database Connector
+
+## Introduction
+
+This guide will walk you through the process of creating a database connector for nodestream. 
```

Comment:
The start of the two sentences is very similar. Maybe the second sentence can start like 
```
We will create a database...
```

Or use a bullet list maybe?